Godeps/_workspace/src/github.com/jbenet/go-peerstream/transport/muxado/muxado.go

```diff
@@ -0,0 +1,80 @@
+package peerstream_muxado
+
+import (
+	"net"
+
+	muxado "github.com/inconshreveable/muxado"
```

Comment:
Oh, it's not rewritten in this PR because it's not used in the codebase (godep only rewrites paths actually included)